src/test/java/cucumber/runner/RunTests.java

```diff
@@ -12,20 +12,39 @@
 import io.cucumber.testng.CucumberOptions;
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeTest;
+import static cucumber.hooks.AccountHooks.deleteAccount;
+import static cucumber.hooks.ContactHooks.deleteContact;
 
 @CucumberOptions(
-        features = {"src/test/resources/features/*"},
-        plugin = {"html:target/site/cucumber-pretty.html", "json:target/cucumber/cucumber.json"},
+        features = {"src\\test\\resources\\feature\\cases"},
```

Comment:
One question, why is in features with \ ?

Reply:
and I think scenarios has its own directories, is this runner for all features? or only cases? or it doesn't have any problem?

Reply:
In the case of windows, the path works with \\ instead of /. The cases was a mistake, I'll delete it, I used it to test only my feature.